### PR TITLE
Saas security url field

### PR DESCRIPTION
**Pull request by eyalpalo**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/43523

## Description
Fixed typo in the incident layout, and added a new incident field.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/PrismaSaasSecurity/IncidentFields/incidentfield-Saas_Security_Asset_URL.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    "associatedTypes": [
+        "Saas Security Incident"
+    ],
+    "associatedToAll": false,
+    "unmapped": false,
+    "unsearchable": false,
+    "caseInsensitive": true,
+    "sla": 0,
+    "threshold": 72,
+    "fromVersion": "5.5.0"
```
Comment (dantavori) [inline Packs/PrismaSaasSecurity/IncidentFields/incidentfield-Saas_Security_Asset_URL.json:30]:
```suggestion
    "fromVersion": "6.0.0"
```

File: Packs/PrismaSaasSecurity/ReleaseNotes/1_2_2.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+- Added new incident field **Saas Security Asset URL** with value **asset_page_uri** from the API.
+
+#### Layouts
+##### Saas Security Layout
+- Added new entry **Asset URL**  to **Remediation Action** section.
+- Fixed typo.
+
+#### Integrations
+##### SaaS Security
+- Removed default classifier.
```
Comment (dantavori) [inline Packs/PrismaSaasSecurity/ReleaseNotes/1_2_2.md:16]:
```suggestion
- Removed the default classifier.
```

File: Packs/PrismaSaasSecurity/ReleaseNotes/1_2_2.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+- **Saas Security Asset URL**
+
+#### Mappers
+##### Saas Security - Incoming Mapper
+- Added new incident field **Saas Security Asset URL** with value **asset_page_uri** from the API.
+
+#### Layouts
+##### Saas Security Layout
+- Added new entry **Asset URL**  to **Remediation Action** section.
+- Fixed typo.
```
Comment (dantavori) [inline Packs/PrismaSaasSecurity/ReleaseNotes/1_2_2.md:12]:
```suggestion
- Fixed a typo.
```

File: Packs/PrismaSaasSecurity/ReleaseNotes/1_2_2.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+#### Incident Fields
+- **Saas Security Asset URL**
+
+#### Mappers
+##### Saas Security - Incoming Mapper
+- Added new incident field **Saas Security Asset URL** with value **asset_page_uri** from the API.
+
+#### Layouts
+##### Saas Security Layout
+- Added new entry **Asset URL**  to **Remediation Action** section.
```
Comment (dantavori) [inline Packs/PrismaSaasSecurity/ReleaseNotes/1_2_2.md:11]:
```suggestion
- Added a new entry **Asset URL** to the **Remediation Action** section.
```

File: Packs/PrismaSaasSecurity/ReleaseNotes/1_2_2.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+
+#### Incident Fields
+- **Saas Security Asset URL**
+
+#### Mappers
+##### Saas Security - Incoming Mapper
+- Added new incident field **Saas Security Asset URL** with value **asset_page_uri** from the API.
```
Comment (dantavori) [inline Packs/PrismaSaasSecurity/ReleaseNotes/1_2_2.md:7]:
```suggestion
- Added a mapping to the **Saas Security Asset URL** incident field.
```

**Review by dantavori — APPROVED** [review]
feel free to merge when fixed

**Comment by bakatzir** [comment]
@eyalpalo can we fix the RN and merge today?

**Pull request merged**